modules/engage-theodul-core/src/main/resources/ui/js/engage/models/meInfo.js

```diff
@@ -69,113 +68,45 @@ define(['jquery', 'backbone'], function($, Backbone) {
-                              matomo_heartbeat = value;
-			    }
-                            else if ((key == prop_matomo_notification)) {
-                              if (value) {
-                              matomo_notification = value;
-                              } else matomo_notification = true; 
-                            }
-                            else if ((key == prop_matomo_track_events) && value) {
-                              matomo_track_events = value;
-                            }
-                        });
-                    }
-                    me.set("allowedtags", allowedTags);
-                    me.set("allowedformats", allowedFormats);
-                    me.set("shortcuts", shortcuts);
-                    me.set("mastervideotype", mastervideotype);
-                    me.set("logo_mediamodule", logo_mediamodule);
-                    me.set("logo_player", logo_player);
-                    me.set("link_mediamodule", link_mediamodule);
-                    me.set("show_embed_links", show_embed_link);
-                    me.set("hide_video_context_menu", hide_video_context_menu);
-                    me.set("shortcut-sequence", shortcut_sequence);
-                    me.set("layout", layout);
-                    me.set("focusedflavor", focusedflavor);
-                    me.set("matomo.server", matomo_server);
-                    me.set("matomo.site_id", matomo_site_id);
-                    me.set("matomo.heartbeat", matomo_heartbeat);
-                    me.set("matomo.notification", matomo_notification);
-                    me.set("matomo.track_events", matomo_track_events);
-                    ready = true;
-                }
-            });
-        },
-        ready: function() {
-            return ready;
+
+                    $.ajax({
+                      url: configURL,
+                      dataType: "text",
+                      success: function(data) {
+                        var rawfile = data;
+                      }
+                    }).then(function(rawfile){
+                      config = jsyaml.load(rawfile);
+                      me.set("config", config);
+                      me.set("allowedtags", config.allowedTags);
+                      me.set("allowedformats", config.allowedFormats);
+                      me.set("shortcuts", config.shortcuts);
+                      me.set("mastervideotype", config.mastervideotype);
+                      me.set("logo_mediamodule", config.logo_mediamodule);
+                      me.set("logo_player", config.logo);
+                      me.set("show_embed_links", config.show_embed_link);
+                      me.set("hide_video_context_menu", config.hide_video_context_menu);
+                      me.set("layout", config.layout);
+                      me.set("focusedflavor", config.focusedflavor);
+                      me.set("matomo.server", config.matomo_server);
```

Comment:
Due to the yaml structure, this should be:

```javascript
if (config.matomo) {
  me.set("matomo.server", config.matomo.server);
  …
  me.set("matomo.track_events", config.matomo.track_events);
}
```

Reply:
I will add that in the code.